scripts/test-in-lxd.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -eux
+
+IMAGE=$1
+SCRIPT=$2
+SERIES=${IMAGE##*:}
+TESTER=mini-iso-tools-$SERIES
+
+lxd init --auto
+
+lxc delete --force $TESTER || true
+
+if [ -z "$(lxc list -f csv -c n ^${TESTER}\$)" ]
+then
+    lxc launch $IMAGE $TESTER
+    lxc config device add $TESTER code disk source=`pwd` path=/src
+else
+    lxc start $TESTER
+fi
+
+lxc exec $TESTER -- sh -ec "
+    cd ~
+    sudo cp -a /src .
```

Comment:
I think sudo is not needed but I guess it does not hurt

Reply:
Confirmed not needed.